src/layouts/base.njk

```diff
@@ -24,7 +24,7 @@
 
     <link rel="shortcut icon" href="{{ config.asset_path | safe }}/blog/assets/img/favicon.ico">
 
-    {%- if not config.is_wordpress_theme %}
+    {%- if not config.is_wordpress_theme and not config.is_404 %}
```

Comment:
Does it matter to have the feed on a 404?

Reply:
I removed it but we could also keep it.